styles/mixins.scss

```diff
@@ -116,3 +116,18 @@
 @mixin button-hover-state($color: --color-black) {
   box-shadow: 0 0 20px var($color);
 }
+
+@mixin wide-background {
+  &::before {
+    background-color: var(--color-neutral-10);
+    content: '';
+    height: 100%;
+    left: 50%;
+    pointer-events: none;
+    position: absolute;
+    top: 0;
+    transform: translate(-50%);
```

Comment:
translateX(-50%) ?

Reply:
To say the truth, I'm not 100% sure as I didn't updated the implementation, just moved it to mixins. I consider it should be just translate.